changelogs/fragments/2329-hiera-lookup-plugin-return-type.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - community.general.hiera lookup plugin - converts the return type of plugin to unicode string (https://github.com/ansible-collections/community.general/pull/2329).
```

Comment:
Good thing that you have the full name ready but this is *the* one context in which we do not use it. These entries get consolidated into this:
https://github.com/ansible-collections/community.general/blob/stable-2/CHANGELOG.rst#v2-5-1

Being already *within* the c.c. repo, it becomes redundant to refer to it over and over again (every single line).

Reply:
Done.